bootstrap.sh

```diff
@@ -35,6 +35,7 @@ usage()
     echo "  -h       Print this help message and exit"
     echo "  +docs    Build also the documentation"
     echo "  +pygelf  Install also the pygelf Python package"
+    echo "  -nodeps  Ignore incompatible Python package dependencies"
```

Comment:
I would use `--no-deps` for POSIX compatibility and move it up in the help message.